source/playbooks/SC/test/security_controls_stack.test.ts

```diff
@@ -1,45 +1,51 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { SynthUtils } from '@aws-cdk/assert';
-import * as cdk from 'aws-cdk-lib';
-import { PlaybookPrimaryStack, PlaybookMemberStack } from '../../../lib/sharrplaybook-construct';
+import { App, DefaultStackSynthesizer, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import {
+  SecurityControlsPlaybookMemberStack,
+  SecurityControlsPlaybookPrimaryStack,
+} from '../lib/security_controls_playbook-construct';
 
-function getTestStack(): cdk.Stack {
-  const app = new cdk.App();
-  const stack = new PlaybookPrimaryStack(app, 'stack', {
+function getTestStack(): Stack {
+  const app = new App();
+  const stack = new SecurityControlsPlaybookPrimaryStack(app, 'stack', {
+    synthesizer: new DefaultStackSynthesizer({ generateBootstrapVersionRule: false }),
     description: 'test;',
     solutionId: 'SO0111',
     solutionVersion: 'v1.1.1',
     solutionDistBucket: 'sharrbukkit',
     solutionDistName: 'aws-security-hub-automated-response-and-remediation',
     remediations: [{ control: 'Example.3' }, { control: 'Example.5' }, { control: 'Example.1' }],
-    securityStandard: 'PCI',
-    securityStandardLongName: 'pci-dss',
-    securityStandardVersion: '3.2.1',
+    securityStandard: 'SC',
+    securityStandardLongName: 'security-control',
+    securityStandardVersion: '2.0.0',
```

Comment:
same for this, it appears to be going to a generic placeholder rather than "real data", but later in this class (ln 40) we're switching from away from generic and using "real data" ?

Reply:
This test was also never running. These values control where it searches for remediation documents. These are the correct values so that it builds real remediation docs. The ones below are still wrong, but don't affect the way the test works because that stack creates different resources. I don't think these tests provide a lot of value so I'm not interested in investing much more time on them at this point.

If and when we get around to building more valuable tests for these stacks, we'll clean these up. That is what I am about to do for the member stack. The purpose of this change is to get us off of a deprecated library so we're not building more things we'll eventually have to migrate.